ci/update-checker/src/main/scala/com/twitter/intellij/updatechecker/UpdateChecker.scala

```diff
@@ -10,8 +10,8 @@ import org.virtuslab.ideprobe.dependencies.IntelliJVersion
 import org.virtuslab.ideprobe.dependencies.Plugin.Versioned
 import pureconfig.generic.auto._
 
-import scala.jdk.CollectionConverters._
-import scala.math.Ordering.Implicits.seqOrdering
+import scala.collection.JavaConverters._
```

Comment:
These are deprecated in Scala 2.13.

Reply:
we need code to be 2.12 and 2.13 compatible. These are 2 lines, won't hurt us and are the. easiest way to make it work.